draft-ietf-quic-multipath.md

```diff
@@ -328,37 +342,85 @@ the PATH_NEW_CONNECTION_ID frame to provide new connection IDs for Path ID 0 and
+consecutive key updates ({{Section 6.5 of QUIC-TLS}}).
+
+When this specification is used, endpoints SHOULD wait for at least three times
+the largest PTO among all the paths before initiating a new key update
+after receiving an acknowledgment that confirms the receipt of the previous key
+update. This interval is different from that in {{QUIC-TLS}}
+which used three times the PTO of the sole single path.
+
+The choice of three times the largest PTO is a trade-off:
+Packets that arrive after their decryption key has been discarded will be dropped.
+Longer delays reduce the probability of losing packets but keeping old keys
+longer can negatively impact the security of the protocol.
+The use of three times the largest PTO aims to minimize packet lost for all paths
+and therefore limits the impact on performance.
+
+Following {{Section 5.4 of QUIC-TLS}}, the Key Phase bit is protected,
+so sending multiple packets with Key Phase bit flipping at the same time
+should not cause linkability issue.
+
 # Path Management {#path-management}
 
 After completing the handshake, endpoints have agreed to enable
 multipath support. They can start using multiple paths when both endpoints
 have issued available connection IDs for at least one unused Path ID.
-If an endpoint receives a disable_active_migration transport parameter
-provided by the peer, it is forbidden to use a new local address
-to establish new paths to the peer's handshake address. However,
-establishment of additional paths from any local address to other peer addresses
-(e.g carried by peer’s preferred_address) is valid immediately.
-
-This document
-does not specify how an endpoint that is reachable via several addresses
-announces these addresses to the other endpoint. In particular, if the
-server uses the preferred_address transport parameter, clients
-cannot assume that the initial server address and the addresses
-contained in this parameter can be simultaneously used for multipath
-({{Section 9.6.2 of QUIC-TRANSPORT}}).
-Furthermore, this document
-does not discuss when a client decides to initiate a new path. We
+This document does not discuss when a client decides to initiate a new path. We
 delegate such discussion to separate documents.
 
-To open a new path, an endpoint MUST use a connection ID associated with
-a new, unused Path ID. To let the peer open a new path, an endpoint needs
-to provide its peer with connection IDs for at least one unused path identifier.
-Still, the receiver may observe a connection ID associated with a used Path ID
-on different 4-tuples due to, e.g., NAT rebinding. In such a case, the receiver reacts
-as specified in {{Section 9.3 of QUIC-TRANSPORT}} by initiating path validation
-but MUST use a new connection ID for the same Path ID.
-
-This proposal adds six multipath control frames for path management:
+This proposal adds six multipath control frames for path management (see Section {{frames}}):
```

Comment:
And what about the PATH_ACK, the PATH_NEW_CONNECTION_ID and the PATH_RETIRE_CONNECTION_ID frames?
```suggestion
This proposal adds nine multipath control frames for path management (see Section {{frames}}):
```